joomla/joomla-cms
# [Guided Tours] Fix dropdown mouse accessibility on fancy selects

### Summary of Changes

When a tour step targets a fancy select box (like the category or the tags field), the dropdown, although accessible with the keyboard, is not accessible with the mouse. Therefore, it is impossible to select a category or a tag via the mouse.

This PR fixes the issue by allowing a target and all inner elements to be available over the tour overlay.

### Testing Instructions

Start the 'Create an article' tour. Go all the way to the selection of a category.
Then try and select tags.
Go through a couple tours to ensure there is no side-effect and that all tours can be run properly.

Note: the z-index of the overlay is 9997, the z-index of the step popups is 9999.
Therefore I chose a z-index of 9998 for the targets. 
A good tour to test z-index values is the welcome tour. In the 'notifications panel' step, step and target can start to overlap on a smaller size window.

### Actual result BEFORE applying this Pull Request

The dropdown cannot be clicked.

![image](https://github.com/user-attachments/assets/11024fb4-ccf2-4ff6-8f57-32dbd43f6e95)

### Expected result AFTER applying this Pull Request

The dropdown can be clicked.

![image](https://github.com/user-attachments/assets/0fb56a65-1476-474e-b89f-3be31d95d3a9)

In the welcome tour:

![image](https://github.com/user-attachments/assets/ff5dba84-d68b-4965-80e3-a317c4b631f8)


### Link to documentations
Please select:
